session/session.go

```diff
@@ -77,6 +97,9 @@ func (s *ZKSession) Subscribe(subscription chan<- ZKSessionEvent) {
 func (s *ZKSession) notifySubscribers(event ZKSessionEvent) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
+	if s.log != nil && event != SessionClosed {
+		s.log.Printf("%v event occured", event)
```

Comment:
Because we can't use e.g. Logrus features for fields here, we should consider posting a little more context, something like:

```go
s.log.Printf("gozk-recipes/session: event occured %s", event.String())
```